harness/determined/core/_tensorboard_mode.py

```diff
@@ -20,3 +20,4 @@ class TensorboardMode(enum.Enum):
 
     AUTO = "AUTO"
     MANUAL = "MANUAL"
+    OFF = "OFF"
```

Comment:
Add docs to the docstring, to match the other ones.

Or let `MANUAL` + `tensorboard_manager=None` suffice.  That's roughly what I had in mind when <maksim? or whoever the author was?> added these other modes.